### ISSUE TITLE
OWM binding crashes after hitting java.lang.IllegalStateException: Not a JSON Object: "{msg: Not found}"

### ISSUE THREAD

**Issue by shutterfreak** [openhab/openhab-addons#6309]
Since a couple days the OWM binding on my OH2.5M4 setup (openHABian Buster on RPi3B+) crashes when the OWM service responds with "{msg: Not found}". After this exception, the binding no longer reports weather updates.

See https://community.openhab.org/t/owm-binding-crashes-after-hitting-java-lang-illegalstateexception-not-a-json-object-msg-not-found/84573 for details.

**Comment by cweitkamp** [comment]
Fix submitted: #6315

**Comment by shutterfreak** [comment]
What is the proper way to upgrade the OWM binding through Karaf to the latest snapshot?
```
openhab> bundle:list | grep -i weather                                                                                                                          
239 │ Active   │  80 │ 2.5.0.M4              │ openHAB Add-ons :: Bundles :: OpenWeatherMap Binding
```
When I run plain `bundle:update 239`, nothing happens beside restarting the current binding version (2.5.0.M4), so I guess I need to find the right URL that points to the (latest shapshot) OWM binding JAR file.
